examples/todo/src/controllers/todo.controller.ts

```diff
@@ -47,6 +47,7 @@ export class TodoController {
     if (todo.remindAtAddress) {
       const geo = await this.geoService.geocode(todo.remindAtAddress);
 
+      /* istanbul ignore next */
       if (!geo[0]) {
```

Comment:
I am not sure if this is the right fix. Can we please add a test to verify handling of this error case instead? Adding "istanbul ignore next" would be ok if such test was too difficult to write.

Reply:
There is a test for this case [here](https://github.com/strongloop/loopback-next/blob/geo-coverage/examples/todo/src/__tests__/acceptance/todo.acceptance.ts#L103), but the issue is when the service is down, it won't run that test, which is causing the coverage to decrease.

Reply:
I see, makes sense 👍🏻

Reply:
@nabdelgadir Can you please add a code comment to capture your explanation for future readers?

Reply:
Added, thanks!